guides/release-notes/v3.0.7.adoc

```diff
@@ -0,0 +1,33 @@
+= Version 3.0.7
+[horizontal]
+Release date:: 2021-05-03
+
+== Changes to BIS document
+* The version numbers used in the BIS-documentation were inconsistant with the actual version numbers listed on the main documentation page. Version numbers should now be aligned. [POACC-391]
+
+*Changes in the documentation of Despatch Advice transaction 3.1. rule T16-R009 and T16-R010 is now reffering to seller party and originator customer party instead of buyer party [POACC-377].
```

Comment:
Order Reference in Dispatch Advice changed cardinality from 1..1 (default) to 0..1, but not mentioned in release notes.